src/main/java/woohakdong/server/api/service/admin/auth/AdminAuthService.java

```diff
@@ -0,0 +1,121 @@
+    private final JWTUtil jwtUtil;
+    private final BCryptPasswordEncoder passwordEncoder;
+
+    @Transactional
+    public LoginResponse login(AdminLoginRequest loginRequest) {
+        Member admin = memberRepository.findByMemberProvideId(loginRequest.username())
+                .orElseThrow(() -> new CustomException(ADMIN_MEMBER_ID_NOT_FOUND));
+
+        if (!passwordEncoder.matches(loginRequest.password(), admin.getMemberPassword())) {
+            throw new CustomException(INVALID_ADMIN_PASSWORD);
+        }
+
+        String access = jwtUtil.createJwt("access", admin.getMemberProvideId(), admin.getMemberRole(), 600000L);
+        String refresh = jwtUtil.createJwt("refresh", admin.getMemberProvideId(), admin.getMemberRole(), 86400000L);
+
+        addRefreshEntity(admin.getMemberProvideId(), refresh, 86400000L);
+
+        return LoginResponse.builder()
+                .accessToken(access)
+                .refreshToken(refresh)
+                .build();
+    }
+
+    @Transactional
+    public void createAdmin(AdminJoinRequest joinRequest) {
+        if (memberRepository.findByMemberProvideId(joinRequest.username()).isPresent()) {
+            throw new CustomException(ADMIN_USERNAME_IS_ALREADY_USED);
+        }
+
+        String encryptedPassword = passwordEncoder.encode("1234");
+
+        Member admin = Member.builder()
+                .memberProvideId(joinRequest.username())
+                .memberName(joinRequest.name())
+                .memberRole("ADMIN_ROLE")
+                .memberEmail(joinRequest.email())
+                .memberPassword(encryptedPassword)
+                .build();
+
+        memberRepository.save(admin);
+    }
+
+    @Transactional
+    public void updateAdmin(AdminInfoUpdateRequest updateRequest) {
+        Member admin = getMemberFromJwtInformation();
+
+        if (memberRepository.findByMemberProvideId(updateRequest.username()).isPresent()) {
+            throw new CustomException(ADMIN_USERNAME_IS_ALREADY_USED);
+        }
+
+        String encryptedPassword = passwordEncoder.encode("1234");
+
+        admin.adminUpdate(updateRequest.username(), updateRequest.name(), updateRequest.email(), encryptedPassword);
+
+        memberRepository.save(admin);
+    }
+
+    public AdminInfoResponse getAdminInfo() {
+        Member admin = getMemberFromJwtInformation();
+
+        return AdminInfoResponse.builder()
+                .username(admin.getMemberProvideId())
+                .name(admin.getMemberName())
+                .email(admin.getMemberEmail())
+                .build();
+    }
+
+    private void addRefreshEntity(String provideId, String refresh, Long expiredMs) {
+
+        Date date = new Date(System.currentTimeMillis() + expiredMs);
+
+        RefreshToken refreshToken = RefreshToken.builder()
+                .refreshProvideId(provideId)
+                .refresh(refresh)
+                .refreshExpiration(date.toString())
+                .build();
```

Comment:
요 친구도 팩토리 메서드 패턴으로 만들어보는건 어떨까요?
```suggestion
        RefreshToken refreshToken = RefreshToken.create(provideId, refresh, date.toString());
```